docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -93,6 +95,10 @@ The restore take a few seconds, after which expect to be logged out.
 
 You should be able to see that all of the data has been restored into the instance, including universes, users, metrics, alerts, task history, cloud providers, and so on.
 
+## Check results
+
+During a high-availability backup, the entire database is copied. If your universes are visible through YugabyteDB Anywhere UI and the replication timestamps are increasing, the backup is successful.
```

Comment:
entire database - confusing. Talking about the YugabyteDB database, or just the Anywhere UI data?

Reply:
Actually, the developer who provided this info said "the entire postgres database". I interpreted it as YugabyteDB database.

Reply:
it is entire postgres database which is what YBA uses to store its state.

To avoid confusion, lets say entire YBA state.

Reply:
changed